Webview add ssl error event

farazs opened the pull request:
## Motivation

When an SSL error occurs in the Android Webview, `onReceivedError` is not called and so there is no event sent to JS side.

Adding a new event allows the Webview to show a specific error message to the user in the case of an SSL error. We still maintain the default behavior of not proceeding with the request.

Additionally, devices with an out of date [Android System Webview](https://play.google.com/store/apps/details?id=com.google.android.webview&hl=en) tend to get this error. This allows showing the user instructions to update this system app when this is the case.

## Test Plan

Manually tested that the `onSslError` method is called on the JS end and the page is not loaded

chirag04 commented on Libraries/Components/WebView/WebView.android.js:
```diff
@@ -264,6 +269,7 @@ class WebView extends React.Component {
         onLoadingStart={this.onLoadingStart}
         onLoadingFinish={this.onLoadingFinish}
         onLoadingError={this.onLoadingError}
+        onLoadingSslError={this.onLoadingSslError}
```
can we fold onLoadingSslError into onLoadingError?

farazs replied:
we would need to be able to distinguish the two though. should we use all the same event code and just add an `isSslError` boolean to the event in `onReceivedSslError`?

farazs commented:
@shergin I've now changed it to use existing `onLoadingError` and added an `isSslError` boolean to the event to be able to distinguish it from other errors.
